src/test/utils.js

```diff
@@ -14,8 +36,38 @@ export const getContext = (state) => {
     runSaga: jest.fn(),
     subscribe: jest.fn(),
     injectedReducers: {},
-    injectedSagas: {}
+    injectedSagas: {},
   };
 
   return { store };
 };
+
+const store = configureStore({}, history);
+
+export const withAppContext = (Component) => (
+  <ThemeProvider>
+    <Provider store={store}>
+      <ConnectedRouter history={history}>
+        {Component}
+      </ConnectedRouter>
+    </Provider>
+  </ThemeProvider>
+);
+
+// eslint-disable-next-line
+export const withCustomAppContext = (Component) => ({ themeCfg = {}, storeCfg = {}, routerCfg = {} }) => (
+  <ThemeProvider {...themeCfg}>
+    <Provider store={store} {...storeCfg}>
+      <ConnectedRouter history={history} {...routerCfg}>
+        {Component}
+      </ConnectedRouter>
+    </Provider>
+  </ThemeProvider>
+);
+
+export const run = (callback) => {
```

Comment:
What purpose does this function serve?

Reply:
hij is weer weg

Reply:
Doesn't look like it's gone ;)

Reply:
kijk je wel in de juiste branch?  bij mij is hij weg